README.md

```diff
@@ -40,12 +40,12 @@ For more information, please refer to the [official documentation](http://danits
 ## Developing the Provider
 If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (version 1.13+ is *required*). You'll also need to correctly setup a [GOPATH](http://golang.org/doc/code.html#GOPATH), as well as adding `$GOPATH/bin` to your `$PATH`.
 
-To compile the provider, run `make build`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+To compile the provider, run `make build`. This will build the provider and put the provider binary in the `./bin` directory.
 
 ```sh
 $ make build
 ...
-$ $GOPATH/bin/terraform-provider-proxmox
+$ ./bin/terraform-provider-proxmox
```

Comment:
Despite me correctly setting `GOPATH` as per the doc, I never saw the binary ending under `$GOPATH/bin`. This is confirmed by looking at the Makefile itself which uses `go build -o "bin/ ..."